.github/workflows/frequency_specs.yml

```diff
@@ -0,0 +1,105 @@
+name: Build Frequency Binary and Specs
+
+on:
+  push:
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9][0-9][0-9]'
```

Comment:
I think we'd want to switch this over to release and then upload the artifacts to release instead of the build.

https://github.com/softprops/action-gh-release